lib/tool_shed/utility_containers/utility_container_manager.py

```diff
@@ -835,10 +825,10 @@ def key_is_current_repositorys_key( self, repository_name, repository_owner, cha
             key_prior_installation_required = components_list[ 4 ]
             key_only_if_compiling_contained_td = components_list[ 5 ]
         if repository_name == key_name and \
-            repository_owner == key_owner and \
-            changeset_revision == key_changeset_revision and \
-            prior_installation_required == key_prior_installation_required and \
-            only_if_compiling_contained_td == key_only_if_compiling_contained_td:
+                repository_owner == key_owner and \
+                changeset_revision == key_changeset_revision and \
+                prior_installation_required == key_prior_installation_required and \
+                only_if_compiling_contained_td == key_only_if_compiling_contained_td:
```

Comment:
Generally speaking, slash continuations should be avoided wherever possible.  Here, I'd say:

```
         if ( repository_name == key_name and
                repository_owner == key_owner and
                changeset_revision == key_changeset_revision and
                prior_installation_required == key_prior_installation_required and
                only_if_compiling_contained_td == key_only_if_compiling_contained_td):
```